src/components/app/details/triggerView/CIMaterialModal.tsx

```diff
@@ -40,8 +62,104 @@ export const CIMaterialModal = ({
         [],
     )
 
-    return (
+    useEffect(() => {
+        if (props.isJobView && props.environmentLists?.length > 0) {
```

Comment:
destructure props

Reply:
Leaving this for now as this will imapct other component where destructuring been used!